.github/actions/run-pre-commit-for-helm/action.yml

```diff
@@ -0,0 +1,17 @@
+name: "Run pre-commit for helm-charts"
```

Comment:
The idea was to avoid duplicating calls to the installation of all the libraries that are needed to check pre-commits for helm charts, which I'm assuming will be always the same. But I missed the part that logic now is split between the pre-commit configuration defined in the repo and the tools that should be installed for that (in build tools), so maybe duplicating a couple of lines it's not a big deal after all. I'm removing this.